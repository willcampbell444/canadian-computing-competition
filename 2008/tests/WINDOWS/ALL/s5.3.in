5
28 29 28 27
28 26 27 29
29 29 29 29
16 27 29 28
22 29 21 19
