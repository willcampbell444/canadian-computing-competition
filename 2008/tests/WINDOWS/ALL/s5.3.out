Patrick
Roland
Patrick
Patrick
Roland
